collector/components/servicenowexporter/go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/lightstep/sn-collector/collector/servicenowexporter
 
-go 1.21
+go 1.21.0
 
-toolchain go1.21.3
+toolchain go1.22.2
 
 require (
 	github.com/stretchr/testify v1.9.0
-	go.opentelemetry.io/collector/component v0.98.0
-	go.opentelemetry.io/collector/config/configopaque v1.5.0
-	go.opentelemetry.io/collector/config/configretry v0.98.0
-	go.opentelemetry.io/collector/exporter v0.98.0
-	go.opentelemetry.io/collector/pdata v1.5.0
-	go.opentelemetry.io/otel/metric v1.25.0
-	go.opentelemetry.io/otel/trace v1.25.0
+	go.opentelemetry.io/collector/component v0.102.1
+	go.opentelemetry.io/collector/config/configopaque v1.9.0
+	go.opentelemetry.io/collector/config/configretry v0.102.1
+	go.opentelemetry.io/collector/exporter v0.102.1
+	go.opentelemetry.io/collector/pdata v1.9.0
+	go.opentelemetry.io/otel/metric v1.27.0
+	go.opentelemetry.io/otel/trace v1.27.0
```

Comment:
It bothers me that go.user.org/zap's version matches OTel-Go's and that testify's version matches stable OTel-Collector's. Looks like very bad luck!